LION/models/learned_regularizer/ACR.py

```diff
@@ -175,44 +173,47 @@ def initialize_weights(self, min_val=0.0, max_val=0.001):
                 self.model_parameters.kernel_size,
                 self.model_parameters.kernel_size,
             ).to(device)
-
+        self.last_layer.weight.data = min_val + (max_val - min_val) * torch.rand_like(
+            self.last_layer.weight.data
+        )
         return self
 
     def forward(self, x):
-
+        # x = fdk(self.op, x)
+        x = self.normalise(x)
         z = self.first_layer(x)
         z = self.first_activation(z)
         for i in range(self.model_parameters.layers):
-            layer = primal_module = getattr(self, f"ICNN_layer_{i}")
+            layer = getattr(self, f"ICNN_layer_{i}")
             z = layer(z, x)
+
         z = self.last_layer(z)
-        return self.pool(z)
+        return self.pool(z).reshape(-1, 1) + self.L2(z)
 
     @staticmethod
     def default_parameters():
-        param = LIONParameter()
-        param.channels = 48
-        param.kernel_size = 5
-        param.stride = 1
-        param.relu_type = "LeakyReLU"
-        param.layers = 10
-        return param
+        return ACRParams(16, 5, 1, "LeakyReLU", 5)
 
     @staticmethod
     def cite(cite_format="MLA"):
         if cite_format == "MLA":
             print("Mukherjee, Subhadip, et al.")
-            print('"Learned convex regularizers for inverse problems."')
-            print("\x1B[3marXiv preprint \x1B[0m")
+            print('"Data-Driven Convex Regularizers for Inverse Problems."')
+            print(
+                "ICASSP 2024-2024 IEEE International Conference on Acoustics, Speech and Signal Processing (ICASSP). IEEE, 2024"
+            )
             print("arXiv:2008.02839 (2020).")
         elif cite_format == "bib":
             string = """
-            @article{mukherjee2020learned,
-            title={Learned convex regularizers for inverse problems},
-            author={Mukherjee, Subhadip and Dittmer, S{\"o}ren and Shumaylov, Zakhar and Lunz, Sebastian and {\"O}ktem, Ozan and Sch{\"o}nlieb, Carola-Bibiane},
-            journal={arXiv preprint arXiv:2008.02839},
-            year={2020}
-            }"""
+            @inproceedings{mukherjee2024data,
+            title={Data-Driven Convex Regularizers for Inverse Problems},
+            author={Mukherjee, S and Dittmer, S and Shumaylov, Z and Lunz, S and {\"O}ktem, O and Sch{\"o}nlieb, C-B},
+            booktitle={ICASSP 2024-2024 IEEE International Conference on Acoustics, Speech and Signal Processing (ICASSP)},
+            pages={13386--13390},
+            year={2024},
+            organization={IEEE}
+            }
+            """
```

Comment:
I would argue the appropiate citation is this:

https://scholar.google.co.uk/scholar?cluster=811116575160518931&hl=en&as_sdt=2005